files/en-us/mdn/mdn_product_advisory_board/members/index.html

```diff
@@ -18,6 +18,9 @@
  <dt><strong>Daniel Appelquist</strong><br>
  Director of Developer Advocacy at Samsung Internet, Co-Chair of W3C's Technical Architecture Group</dt>
  <dd>Daniel Appelquist leads the Web Developer Advocacy team at Samsung Internet &lt;<a href="https://samsunginter.net">https://samsunginter.net</a>&gt;, Samsung's mobile and VR web browser. He has also been co-chair of the <a href="https://w3.org/tag">W3C's Technical Architecture Group</a> since 2013. He has been working on the web for over 2 decades, first helping to create start-ups in the web publishing sphere and then (after moving to London from New York in '99) working on the convergence between web and mobile through many projects and initiatives. He has also led open standards strategy for the UK Government and remains a member of the <a href="https://www.gov.uk/government/groups/open-standards-board">UK Government Open Standards Board</a>. In between all this he's been an active community builder and event organizer.</dd>
+ <dt><strong>Sukriti Chadha</strong><br>
+ <strong>Product Manager, Spotify</strong></dt>
+ <dd>Sukriti Chadha is a mobile developer-turned-product manager at Spotify where she leads efforts in cross-platform accessibility and developer experience on mobile. She is also an Invited Expert at the <a href="#">W3C WCAG (Web Content Accessibility Guidelines) Working Group</a> and the <a href="#">Mobile Accessibility Task Force (MATF)</a>, where she contributes to industry guidelines for accessible web and mobile applications. She built and patented a new way of making data visualization accessible, and <a href="#">open-sourced the Android solution in 2019</a>. Sukriti’s product role entails managing developer experience roadmaps, automated testing, and embedding accessibility in the product lifecycle.</dd>
```

Comment:
@scha14 copied your changes as a suggestion to the existing PR
```suggestion
 <dd>Sukriti Chadha is a mobile developer-turned-product manager at Spotify where she leads efforts in cross-platform accessibility and developer experience on mobile. She also serves an Invited Expert at the <a href="https://www.w3.org/WAI/standards-guidelines/wcag/">W3C WCAG (Web Content Accessibility Guidelines) Working Group</a> and the <a href="https://www.w3.org/WAI/standards-guidelines/mobile/">Mobile Accessibility Task Force (MATF)</a>, where she contributes to industry guidelines for accessible web and mobile applications. She built a new way of making data visualization accessible for viusally impaired users, and <a href="https://developer.yahoo.com/blogs/612790529269366784/">open-sourced the Android solution in 2019</a>. Sukriti’s product role entails managing developer experience roadmaps, automated testing, and embedding accessibility in the product lifecycle.</dd>
```

Reply:
Thank you @nschonni

Reply:
No problem, feel free to make the same suggestion here to get the "co-authored" credit. I did just change "serves as" instead of "serves"

Reply:
Thank you so much! This looks great 👍